app.json

```diff
@@ -12,7 +12,7 @@
         "required": true
```

Comment:
As we are moving over to Github tokens and OAuth, shouldn't app.json be removed?

Reply:
This provides a streamlined interface for deploying a new app.

Reply:
but this allows them to create Heroku apps without linking to Github which screws up with some commands